doc/nle/source/getting_started.rst

```diff
@@ -54,7 +55,18 @@ Optionally, one can clone the repository and install the package manually.
 Trying it out
 *************
 
-NLE comes with a few scripts that allow to get some environment rollouts, and
+After installation, one can try out any of the provided tasks as follows:
+
+.. code-block:: python
+    
+    import gymnasium as gym
+    import nle
+    env = gym.make("NetHackScore-v0")
+    env.reset()  # each reset generates a new dungeon
+    env.step(1)  # move agent '@' north
```

Comment:
Is `1` intuitive here? Or should we instead do a variable lookup?

Reply:
Those lines are a direct copy of the code in README.md. 

It is a good suggestion. In the test I added for truncation, it uses this step formulation:

`obs, reward, termination, truncation, info = env.step(nethack.MiscDirection.WAIT)`

rather than the ordinal number of the WAIT action. But it would be nice to have something on the lines of:

```
from nethack import actions
...
env.step(actions['WAIT'])
```

Sounds like a Request for Enhancement :)

Reply:
Agreed we should have one enum with all actions. Kinda silly we don't, actually. One added complexity is that the RL actions are `0` to `N-1` while the underlying NetHack actions are keyboard commands where e.g. `WAIT` is space. (Another problem is that actions change their meaning given the context, e.g. for itemized lists.)